lte/gateway/configs/redis_for_container.conf

```diff
@@ -0,0 +1,33 @@
+#
+# Copyright (c) 2016-present, Facebook, Inc.
```

Comment:
Can we move all the container related configs into lte/gateway/configs/docker/ sub-directory? Currently it is split across lte/gateway/configs and lte/gateway/configs/templates/ ?

Reply:
fixed